### PR TITLE
Feat/createp

### LINKED ISSUE #6: Create toggle function

### PR DESCRIPTION
**Pull request by MaxBB-RC**
closes #6

### CODE REVIEW

File: api/sft/handler.go [hunk 1]
```diff
@@ -39,3 +47,57 @@ func (h *Handler) GetAllToggles(w http.ResponseWriter, r *http.Request) {
 
 	utils.WriteJSON(w, toggles)
 }
+
+func (h *Handler) CreateToggle(w http.ResponseWriter, r *http.Request) {
+	fmt.Println("creating toggle")
```
Comment (revitteth) [inline api/sft/handler.go]: Would be good to use log package here - @elliothllm wants to abstract it to its own repo, and we probably should fmt.Println will output to stderr all the time - it's not as controllable as having log.Logger().Trace or .Debug for example where we can exclude it easily from server logs.

File: api/sft/service.go [hunk 2]
```diff
@@ -21,13 +23,13 @@ type Service struct {
...
 func NewService(db ISimpleFeatureToggleDb, ctx context.Context, pool *pgxpool.Pool) *Service {
 	fmt.Println("new service!")
-	if err := RunDbMigrations(ctx, pool); err != nil {
-		panic(err)
-	}
-
-	if err := RunDbSeed(ctx, pool); err != nil {
-		log.Println("db seed failed")
-	}
+	//if err := RunDbMigrations(ctx, pool); err != nil {
```
Comment (revitteth) [inline api/sft/service.go]: Commented code can either be used or removed - tend to prefer to use git to reclaim code vs comments.

File: api/tests/create_toggle_test.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	}
+
+	// create a duplicate entry to test against.
+	initialToggle, err := sftService.CreateToggle(context.Background(), duplicateToggle)
+	if err != nil {
+		t.Errorf("error creating initial toggle: %s", err)
+	}
+	_ = initialToggle
+
+	var tests = []struct {
```
Comment (revitteth) [inline api/tests/create_toggle_test.go]: I would tend to call this 'scenarios' and make it map[string]struct{} - that way the string key for the map becomes the 'test name' and goland and the test runner will react to it as such (useful tip!)

File: api/sft/handler.go (hunk 1 above)
Comment (MaxBB-RC) [reply api/sft/handler.go]: Cool, have changed to log.Println for now.

File: api/sft/service.go (hunk 2 above)
Comment (MaxBB-RC) [reply api/sft/service.go]: Removed.

File: api/tests/create_toggle_test.go (hunk 3 above)
Comment (MaxBB-RC) [reply api/tests/create_toggle_test.go]: That is a nice tip! I've changed the tests accordingly.

**Pull request merged**